pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -2949,6 +2949,55 @@ func TestContextValid(t *testing.T) {
 					Name: "a-param-mat", Value: ParamValue{ArrayVal: []string{"$(context.pipelineTask.retries)"}},
 				}}},
 		}},
+	}, {
+		name: "valid string context variable for Pipeline name in include params",
+		tasks: []PipelineTask{{
+			Name:    "bar",
+			TaskRef: &TaskRef{Name: "bar-task"},
+			Params: []Param{{
```

Comment:
Make sure to keep v1beta1 in sync with v1 (i.e. copy over the changes made in v1 tests)